ia-wishlist-bot/get_wishlist_editions_isbn.py

```diff
@@ -0,0 +1,38 @@
+"""
+Python Script to get a list of ISBNs which aren't added to Open Library
+
+Input: 'wishlist_works_edition.ndjson'
+* Contains the parameters "oclc_synonyms",  "openlibrary_synonyms", "editions" -> {"isbn", "olid"}
+
+Output: 'ol_works.csv'
+* Contains the Parameter: 'isbn-13' for all the books which aren't added to Open Library
+"""
+
+import ndjson
+import csv
+import time
+
+new_data = []
+
+
+with open('/storage/openlibrary/wishlist/wishlist_works_editions.ndjson') as f:
+	data = ndjson.load(f)
+
+start_time = time.time()
+
+for i in range(len(data)):
+	flag = 0
+	if len(data[i]['openlibrary_synonyms']) == 0:
+		for edition in data[i]['editions']:
+			if edition['olid'] == None:
```

Comment:
values should not be `==` compared to singletons like `None` or `True`. Use `is` for these cases:

`if edition['olid'] is None`

See PEP 8 (https://legacy.python.org/dev/peps/pep-0008/#id48) and the justifications here https://stackoverflow.com/questions/25317175/why-comparisons-to-singletons-like-none-should-always-be-done-with-is-or-is-not and for reasons why.